src/librustc_codegen_llvm/back/write.rs

```diff
@@ -422,7 +420,7 @@ pub(crate) unsafe fn optimize(cgcx: &CodegenContext<LlvmCodegenBackend>,
                  || {
             llvm::LLVMRustRunFunctionPassManager(fpm, llmod)
         });
-        timeline.record("fpm");
+        cgcx.profile(|p| p.generic_event("fpm"));
         time_ext(config.time_passes,
                  None,
                  &format!("llvm module passes [{}]", module_name.unwrap()),
```

Comment:
This `time_ext` call should also have `start-end` activity called `module passes` or something like that.